csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -872,8 +872,14 @@ std::shared_ptr<ReductionParams> innerOuterPersistentHeuristic(
       rparams->pad_outer_reduction_to_warp = true;
     }
     rparams->block_dim_iter_dom = ParallelType::TIDy;
+    rparams->combined_split_grid_inner_dim =
+        iop.vectorization_factor_outer * iop.bdimy * iop.gdimy <
+        inner_dim_numel;
   } else {
     rparams->block_dim_inner_reduction_extra = ParallelType::TIDy;
+    rparams->combined_split_grid_inner_dim =
+        iop.vectorization_factor_outer * iop.bdimx * iop.gdimy <
+        inner_dim_numel;
```

Comment:
nitpic: IIUC, the above block is identical for the if-else-blocks. maybe just move it out so we don't have redundancy

Reply:
They are slightly different:
```
if(){
vect * bdimy * gdimy < inner_dim
}else{
vect * bdimx * gdimy < innder_dim
}
```

Reply:
ah, sorry about that. 🙇